commit/merkleroot/observation.go

```diff
@@ -74,18 +81,14 @@ type Observer interface {
 	// ObserveMerkleRoots computes the merkle roots for the given sequence number ranges
 	ObserveMerkleRoots(ctx context.Context, ranges []plugintypes.ChainRange) []cciptypes.MerkleRootChain
 
-	ObserveTokenPrices(ctx context.Context) []cciptypes.TokenPrice
-
-	ObserveGasPrices(ctx context.Context) []cciptypes.GasPriceChain
-
 	ObserveFChain() map[cciptypes.ChainSelector]int
 }
 
 type ObserverImpl struct {
 	lggr         logger.Logger
 	homeChain    reader.HomeChain
 	nodeID       commontypes.OracleID
-	chainSupport ChainSupport
+	chainSupport shared.ChainSupport
```

Comment:
`shared` is a very generic package name. Let's use something more go idiomatic

Reply:
`utils`? :D